pcawg_metadata_parser/generate_QC_reports.py

```diff
@@ -596,6 +596,57 @@
      }
 },
 
+#query 11: get donors having fixed or version higher than 1.0.4 DKFZ/EMBL
+{
+  "name": "dkfz_embl_merged_and_higher_versions",
```

Comment:
The name does not quite make sense. Can we say something like this? *dkfz_embl_single_gnos_uploads*